QEfficient/cloud/infer.py

```diff
@@ -119,6 +134,41 @@ def main(
         full_batch_size=full_batch_size,
     )
 
+    #########
+    # Log
+    #########
+
+    if benchmark:
+        input_len = max([len(x) for x in tokenizer(prompt, return_tensors="np").input_ids])
+        num_chunks = -(input_len // -prompt_len)
+        input_len = num_chunks * prompt_len
+
+        fields = {
+            "MODEL\nNAME": model_name,
+            "BATCH\nSIZE": batch_size,
+            "FULL\nBATCH_SIZE": full_batch_size,
+            "CPL": prompt_len,
+            "PL": input_len,
+            "GL": generation_len,
```

Comment:
You can return total_decoded_tokens from regular_model_execution and continuous_batching_execution functions in Class TextGeneration and save in CloudAI100ExecInfo in QEfficient/generation/text_generation_interface.py and here GL will be execinfo.total_decoded_tokens + 1

Reply:
Thanks for the guideline. Did exactly as you said and exported the total_decoded_tokens to the execinfo. Please see the commit https://github.com/quic-morteza/efficient-transformers/commit/8f6abc5a57bd369c4deb1cea8d18725ddc57e2a2#diff-b0eeaff0b880369081d78debfcafcc8b62e14f67f4b0cbf2a02d506284131bc9